piglit/README.md

```diff
@@ -0,0 +1,38 @@
+# piglit snap
+
+This project answers the musical question
+"How well does OpenGL work inside a snap?"
+
+It also illustrates how to build up a snap from
+three pieces: one for upstream python bits, one for
+upstream C bits, and one for a wrapper.
+
+It uses the log-observe plug so it can access dmesg
+(so it can tell which test cases cause kernel error messages).
+
+Finally, comments in snapcraft.yaml illustrate each kludge along
+the way to the final gargantuan commandline (which probably
+should be replaced with a small wrapper script specific to
+this project).
+
+Example usage:
+
+First, do a very quick sanity check to make sure you can access the graphics card:
+
+$ piglit run tests/sanity results/sanity
+$ piglit summary console results/sanity
+
```

Comment:
Where are these tests shipped to? Can we note down the path here? Even if it's something like `/snap/piglit/current/<someplace>`?